src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -34,15 +36,15 @@
 
 	private static final long serialVersionUID = 1L;
 
-	private final List<AssertionError> failures = new ArrayList<AssertionError>();
+	private final List<Throwable> failures = new ArrayList<Throwable>();
 
 	private final String heading;
 
 	public MultipleFailuresError(String heading) {
 		this.heading = isBlank(heading) ? "Multiple Failures" : heading.trim();
 	}
 
-	public void addFailure(AssertionError failure) {
+	public void addFailure(Throwable failure) {
```

Comment:
Well, changing the signature of `getFailures()` breaks existing consumers, e.g. IDEs, in case they use this API. Removing `addFailure()` on the other hand "only" breaks test engines, probably only Jupiter, so it should be okay while we're still doing milestone releases.